instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go

```diff
@@ -498,6 +498,11 @@ func StreamServerInterceptor(opts ...Option) grpc.StreamServerInterceptor {
 	}
 }
 
+func metricAttributes(fullMethod string) []attribute.KeyValue {
+	_, attrs := internal.ParseFullMethod(fullMethod)
```

Comment:
This is now being called twice. Once in `spanInfo` and now here. Why not just update `spanInfo` to returns two sets of attributes, one for spans and the other for metrics. The latter being the equivalent of what is returned here?